src/Runner.Listener/JobDispatcher.cs

```diff
@@ -512,14 +512,14 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                             var completedTask = await Task.WhenAny(renewJobRequest, workerProcessTask, Task.Delay(-1, jobRequestCancellationToken));
                             if (completedTask == workerProcessTask)
                             {
+                                var detailInfo = string.Join(Environment.NewLine, workerOutput);
+                                Trace.Info(detailInfo);
```

Comment:
`Trace.Info` formats the input still - the first line would look like

`[RUNNER YYYY-MM-DD:HH:mm:ss TYPE] [WORKER YYYY`

Would not mislead a human, but it can trip up some automation